VS/cudaDeconv.sln

```diff
@@ -1,140 +1,30 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ALL_BUILD", "ALL_BUILD.vcxproj", "{2BFB8219-101C-3EED-BD02-D3DD1AD9DFAF}"
-	ProjectSection(ProjectDependencies) = postProject
-		{A664BE1E-37ED-394C-B18C-FE874C807DD0} = {A664BE1E-37ED-394C-B18C-FE874C807DD0}
-		{B16BA03E-F6FF-3452-A6EC-609C86A17CF5} = {B16BA03E-F6FF-3452-A6EC-609C86A17CF5}
-		{18735949-1B01-34B9-9EAE-238D53D9EA1C} = {18735949-1B01-34B9-9EAE-238D53D9EA1C}
-		{42454AEF-8A2A-3F9B-A4D2-86EB2AC6B42D} = {42454AEF-8A2A-3F9B-A4D2-86EB2AC6B42D}
-		{2F3F7C90-6CCC-3257-9AE1-12F5243B428C} = {2F3F7C90-6CCC-3257-9AE1-12F5243B428C}
-		{4F6106FA-C97C-3A9F-9450-C7A9D72CD5B4} = {4F6106FA-C97C-3A9F-9450-C7A9D72CD5B4}
-		{5F27E56C-C314-33A3-840D-5F68852D364B} = {5F27E56C-C314-33A3-840D-5F68852D364B}
-	EndProjectSection
-EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Buffer", "Buffers\Buffer.vcxproj", "{A664BE1E-37ED-394C-B18C-FE874C807DD0}"
-	ProjectSection(ProjectDependencies) = postProject
-		{18735949-1B01-34B9-9EAE-238D53D9EA1C} = {18735949-1B01-34B9-9EAE-238D53D9EA1C}
-	EndProjectSection
-EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "INSTALL", "INSTALL.vcxproj", "{01610001-4D16-313E-B831-0DD4211AC2B6}"
-	ProjectSection(ProjectDependencies) = postProject
-		{2BFB8219-101C-3EED-BD02-D3DD1AD9DFAF} = {2BFB8219-101C-3EED-BD02-D3DD1AD9DFAF}
-		{18735949-1B01-34B9-9EAE-238D53D9EA1C} = {18735949-1B01-34B9-9EAE-238D53D9EA1C}
-	EndProjectSection
-EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "RL_interface_driver", "RL_interface_driver.vcxproj", "{B16BA03E-F6FF-3452-A6EC-609C86A17CF5}"
-	ProjectSection(ProjectDependencies) = postProject
-		{A664BE1E-37ED-394C-B18C-FE874C807DD0} = {A664BE1E-37ED-394C-B18C-FE874C807DD0}
-		{18735949-1B01-34B9-9EAE-238D53D9EA1C} = {18735949-1B01-34B9-9EAE-238D53D9EA1C}
-		{2F3F7C90-6CCC-3257-9AE1-12F5243B428C} = {2F3F7C90-6CCC-3257-9AE1-12F5243B428C}
-	EndProjectSection
-EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ZERO_CHECK", "ZERO_CHECK.vcxproj", "{18735949-1B01-34B9-9EAE-238D53D9EA1C}"
-	ProjectSection(ProjectDependencies) = postProject
-	EndProjectSection
-EndProject
+VisualStudioVersion = 15.0.28307.329
+MinimumVisualStudioVersion = 10.0.40219.1
```

Comment:
I've never used the `.sln` files, so I would just mention @linshaova here in case this has the potential to change his local build process?

Reply:
The existing .sln file was pretty broken, unlinked to anything.  I'm only using .sln just for the Visual Studio IDE, specifically intellisense (so I can find references to functions, etc.).  I'd welcome any changes to the .sln as I'm just hacking away at VS settings with no knowledge of what best practices are (e.g. there are probably more hard-coded paths, that should be relative paths, etc.)

Tangentially, I believe the main build system should be the conda recipe stuff.  It's a turn-key build system and we should all continue to use and exercise it.

Reply:
for what it's worth, if you haven't tried it, [VSCode](https://code.visualstudio.com/) is a pretty awesome editor.  the [c/c++ extension](https://code.visualstudio.com/docs/languages/cpp) will give you all the intellisense features and i find the editor to be much more enjoyable (if you're not using the full build chain of Visual Studio)...